ecl/eclcc/eclcc.cpp

```diff
@@ -2575,6 +2604,10 @@ int EclCC::parseCommandLineOptions(int argc, const char* argv[])
             else
                 optMetaLocation.clear();
         }
+        else if (startsWith(arg, "--neverSimplify="))
```

Comment:
@shamser A comment seems to have got lost.
This should be using iter.matchOption(), otherwise it is inconsistent (and would not support --neverSimplify x.y).